plugins/katello/nightly/user_guide/remote_databases/index.md

```diff
@@ -24,6 +24,14 @@ There are two ways to deploy Foreman with remote databases:
     * restore the DBs on remote servers
     * run foreman-installer with right parameters pointing to the databases. It re-configures the databases and start all the services with new DB locations
 
+### Important notes:
+
+* Pulpcore installs a separate PostgreSQL database. Databases should be upgraded to avoid conflicts.
+
+* Katello 3.16 added dependencies provided by the `rh-postgresql12-postgresql-evr` package. This should be installed on any external PostgreSQL v12 database before applying any db migrations.
+
+Note that [Foreman v2.0 requires PostgreSQL v10+](https://theforeman.org/manuals/2.0/index.html#Headlinefeatures) and users are encouraged to upagrade to PostgreSQL v12.
```

Comment:
Typo: "upagrade"

Will the postgresql12-postgresql-evr note also be added to the 3.18, 3.17, and 3.16 docs pages?

Reply:
@ianballou thank you for spotting that typo. I believe we had evr notes for previous version, but decided to remove them.